force_gromacs/io/base_file_reader.py

```diff
@@ -0,0 +1,60 @@
+from traits.api import HasTraits, ReadOnly
+
+
+class BaseFileReader(HasTraits):
```

Comment:
Base file reader class that requires the ability to read simple text files (`_read_file` method) and parse them for relevant information (`_get_data` method).  Calling `read` with a given file path will return parsed information in a form required by the child class.